src/app/viam-client.ts

```diff
@@ -34,6 +35,7 @@ export class ViamClient {
   private serviceHost: string;
 
   public dataClient: DataClient | undefined;
+  public appClient: AppClient | undefined;
```

Comment:
The `viam-client.test.ts` file also has tests to show that the ViamClient has these created when the `createViamClient()` function is called. 
`expect(client.appClient).toBeInstanceOf(AppClient);`